File: Python Excel Examples/WorksheetsApiDemo/replaceText.py
import spirecloudexcel
from spirecloudexcel.configuration import Configuration as ExcelConfiguration

appId = "your id"
appKey = "your key"
baseUrl="https://api.e-iceblue.cn"
configuration = ExcelConfiguration(appId, appKey,baseUrl)
api = spirecloudexcel.api.worksheets_api.WorksheetsApi(configuration)

name = "ReplaceText.xlsx"
folder = "ExcelDocument";
storage = ""
sheetName = "Sheet1"
oldValue = "North America"
newValue = "北美"
matchCase = 'true'
api.replace_text(name, sheetName, oldValue,newValue, match_case = matchCase,folder=folder, storage=storage)

File: PHP Excel Examples/WorksheetsApiDemo/replaceText.php
<?php

use Spire\Cloud\Excel\Sdk\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey, $baseUrl);
$apiInstance = new WorksheetsApi($configuration);

$name = "ReplaceText.xlsx";
$folder = "input";
$storage = null;
$sheetName = "Sheet4";
$oldValue = "North America";
$newValue = "USA";
#default value is false
$matchCase = true;
$result = $apiInstance->replaceText($name,  $sheetName, $oldValue, $newValue,$matchCase,$folder, $storage);